__tests__/app.test.js

```diff
@@ -47,3 +47,60 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("should return an array of comments with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        response.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("should return all comments for a specified article", () => {
```

Comment:
I can see you've dealt with some of the above comments here in a separate test. This isn't a separate behaviour so no need for it to be a separate test. If you make the changes above you can remove this test.